packages/nx/src/command-line/init/implementation/utils.ts

```diff
@@ -214,11 +214,15 @@ export function printFinalMessage({
   const normalizedBodyLines = (bodyLines ?? []).map((l) =>
     l.startsWith('- ') ? l : `- ${l}`
   );
+  const pmc = getPackageManagerCommand();
 
   output.success({
     title: '🎉 Done!',
     bodyLines: [
-      '- Enabled computation caching!',
+      `- Run "${pmc.exec} nx run-many --t build" to run the build target for every project in the workspace. Run it again to replay the cached computation.`,
+      `  - https://nx.dev/features/cache-task-results`,
+      `- Run "${pmc.exec} nx graph" to see the graph of the monorepo. It will show tasks that you can run with Nx.`,
```

Comment:
`graph of the monorepo` - it may be a standalone workspace

Maybe reword to `Run "${pmc.exec} nx graph" to explore the projects and tasks in your workspace.